static/docs/commands-reference/repro.md

```diff
@@ -15,25 +15,83 @@ name is omitted, `Dvcfile` will be used by default.
-        -m, --metrics         Show metrics after reproduction
-        --dry                 Only print the commands that would be executed without
-                              actually executing.
-        -i, --interactive     Ask for confirmation before reproducing each stage.
-        -p, --pipeline        Reproduce the whole pipeline that the specified stage
-                              file belongs to.
 ```
 
+
+## Description
+
+`dvc repro` provides an interface to rerun the commands in the 
+computational graph (a.k.a. pipeline) defined by the stage files 
+in the current workspace.  By default, this command searches, 
+starting from the `Dvcfile`, the pipeline stages to find any
+which have changed.  It then reruns the corresponding commands.  The
+pipeline is defined using the `dvc run` command.
+
+There are several ways to limit the stages to rerun:
+
+* Listing stage file(s) as `targets`
+* The `--single-item` option ensures that only a single stage is rerun.
+* The `--pipeline` option limits execution to the pipeline
+ containing the named stage, especially useful if the
+ workspace contains multiple pipelines.  
+* The `--cwd` option limits execution to the pipeline 
+ located in a subdirectory.
+
+There are two ways to ensure rerunning additional stages beyond
+a changed stage:
+
+* The `--force` option causes the entire pipeline to be rerun
+* The `--ignore-build-cache` causes all stages following a changed stage
+ to be rerun.
+
 `dvc repro` does not run `dvc fetch`, `dvc pull` or `dvc checkout` to get source
 data files, intermediate or final results.
 
+## Options
+
+* `-q`, `--quiet` do not write anything to standard output.  The command
+  run by the stage is free to make output irregardless of this flag.
+  Exit with 0 if all stages are up to date or if all stages
+  are successfully rerun, otherwise exit with 1. 
+
+* `-v`, `--verbose` displays detailed tracing information from executing the
+  `dvc repro` command.
+
+* `-f`, `--force`  Rerun the pipeline, reproducing its results, even
+ if no changes were found.  By default this reruns the entire pipeline.
+ To rerun a single stage, specify the stage name on the command-line
+ along with the `--single-item` option.
+
+* `-s`, `--single-item`  Reproduce only a single data item without 
+ recursively checking for changed dependencies.
+ Multiple stages are rerun if multiple stage names are listed on the command-line.
+
+* `-c`, `--cwd`  Directory within your project to reproduce from.  If no 
+ target names are given, it attempts to use `Dvcfile`, if it exists
+ in the named directory, for stages to rerun.
+ Instead of using `--cwd` one can alternately specify a target in
+ a subdirectory as `path/to/target.dvc`.
+
+* `-m`, `--metrics`  Show metrics after reproduction.  The pipeline must
+ have at least one metrics file defined either with the `dvc metrics` command,
+ or by the `--metrics-no-cach` option on the `dvc run` command.
```

Comment:
there is also a second option `--metrics`. I would actually mention their short counterparts - `-M` and `-m`

Reply:
Have updated the file to match